assets/src/common/helpers/index.js

```diff
@@ -392,3 +392,35 @@ export const getContentLengthFromUrl = async ( url ) => {
 	} );
 	return Number( response.headers.get( 'content-length' ) );
 };
+
+/**
+ * Sets the featured image, on selecting it in the Media Library.
+ *
+ * @param {string} url             Image URL.
+ * @param {number} id    Attachment ID.
+ * @param {number} width           Image width.
+ * @param {number} height          Image height.
+ * @param {Function} onSelect      A function in the MediaUpload component called on selecting the image.
+ * @param {Function} dispatchImage A function to dispatch the change in image to the store.
+ */
+export const setImageFromURL = ( { url, id, width, height, onSelect, dispatchImage } ) => {
```

Comment:
This is copied [almost verbatim](https://github.com/ampproject/amp-wp/blob/bcda6d639920c8fe5c5e5fa5af83ef15d815a72f/assets/src/common/components/with-cropped-featured-image.js#L259). It's in a separate function so 2 classes can use it.